release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1051,61 +1355,371 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+
+* Previously, during an upgrade, the Machine Config Controller (MCC) for the previous release could react to a configuration change from the newer Machine Config Operator (MCO). The MMC then introduced another change that resulted in an unnecessary reboot during the upgrade process. This bug fix prevents the MCC from reacting to a configuration change from a newer MCO and avoids an unnecessary reboot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1879099[*BZ#1879099*])
+
+* Previously, the forward plugin for CoreDNS distributed queries randomly to all the configured DNS servers. Name resolution failed intermittently because CoreDNS would query a DNS server that was not functional. This bug fix sets the forward plugin to use the sequential policy so that queries are sent to the first DNS server that is responsive. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1882209[*BZ#1882209*])
+
+* Previously, the Machine Config Operator was reading enabled systemd target units only from the `multi-user.target.wants` directroy. As a consequence, any unit that does not target the `multi-user.target.wats` directory was changed to target that directory. This fix modified the MCO to use the systemd-preset file to create a preset file in the MCO. As a result, all systemd services are enabled and disabled as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1885365[*BZ#1885365*])
+
+* Previously, when migrating a cluster to the OVN-Kubernetes default Container Network Interface (CNI), bond options on a pre-configured Linux bond interface. As a consequence, bonds are configured using round-robin instead of the mode specified and the bonds might not function. The ovs-configuration.service (configure-ovs.sh) was modified to copy all of the previous bond options on the Linux bond to `ovs-if-phys0` Network Manager connection. As a result, all bonds should work as originally configured. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899350[*BZ#1899350*])
+
+* In {product-title} 4.6, a change was made to use the BFQ (Budget Fair Queueing) Linux I/O scheduler. As a consequence, there was an increased fsync I/O latency in etcd. This fix modified the I/O scheduler to use the mq-deadline scheduler (except for NVMe devices, which are configured to not use an I/O scheduler). For {op-system-first} updates, the BFQ scheduler is still used. As a result, latency times have been reduced to acceptable levels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899600[*BZ#1899600*])
+
+*Web console (Administrator perspective)*
+
+* Previously, an issue with a dependency, resulted in the persistent unmounting and remounting of the YAML Editor in the {product-title} web console. As a consequence, the YAML editor jumped to the top of the YAML file every few seconds This fix removed a default parameter value for the dependency. As a result, the YAML Editor behaves as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1903164[*BZ#1903164*])
```

Comment:
YAML Editor or YAML editor? And is it a UI element?

Reply:
UI element; I've formatted accordingly